imports/plugins/included/jobcontrol/server/jobs/exchangerates.js

```diff
@@ -22,13 +22,23 @@ export function setupFetchFlushCurrencyHooks() {
           backoff: "exponential" // delay by twice as long for each subsequent retry
         })
         .repeat({
-          schedule: later.parse.text(refreshPeriod)
+          schedule: Jobs.later.parse.text(refreshPeriod)
         })
         .save({
           // Cancel any jobs of the same type,
           // but only if this job repeats forever.
           cancelRepeats: true
         });
+
+      // Run the first time immediately after server start. The repeat({schedule}) option via
+      // later.js scheduling won't run before the first scheduled time, even with delay() or after()
+      new Job(Jobs, "shop/fetchCurrencyRates", {})
+        .priority("normal")
```

Comment:
Not a big deal, but `.priority("normal")` is the default, so this is redundant/unnecessary to have here.

Reply:
done